fe1-web/src/features/lao/network/LaoHandler.ts

```diff
@@ -96,10 +103,7 @@ export const handleLaoGreetMessage = (msg: ProcessableMessage): boolean => {
     // return true;
   }
 
-  // add the lao#greet message to the store to tell LaoGreetWatcher to keep an eye on it
-  dispatch(addGreetLaoMessage({ laoId: msg.laoId.valueOf(), messageId: msg.message_id.valueOf() }));
-
-  handleLaoGreet(greetLaoMsg, msg.sender);
+  handleLaoGreet(msg.message_id, greetLaoMsg, msg.sender);
```

Comment:
Hmm yeah I should for sure add a tsdoc statement to state what the function does. I am not sure about the name `connectToPeerBackends` since the function also stores the server for the respective lao (including the backend's public key) but I will try to think of a better name!